contracts/netmap/contract.go

```diff
@@ -61,6 +79,10 @@ const (
 	newEpochSubscribersPrefix = "e"
 	cleanupEpochMethod        = "newEpoch"
 
+	// Node2 storage.
+	node2CandidatePrefix = "2"
```

Comment:
It can be `q`, `w`, whatever. The main requirement is to not have any intersections with other keys. If it has any relation to meaning --- ok. Currently it has, `c` can't be used, `1` won't mean anything at all.

Reply:
idk, letter prefixes can be useful imo, this is the first num prefix as i remember